### PR TITLE
Alfred, don't do anything

### PR DESCRIPTION
**Pull request by stlaz**
Stop serving.

### CODE REVIEW

**Comment by stlaz** [comment]
/hold
Hold until the Service CA operator becomes managed.

**Comment by enj** [comment]
https://github.com/openshift/service-ca-operator/pull/22 will merge soon.

/lgtm
/retest

File: pkg/operator/operator.go [hunk 1]
```diff
@@ -94,6 +94,9 @@ func (c serviceCertSignerOperator) Key() (metav1.Object, error) {
 }
 
 func (c serviceCertSignerOperator) Sync(obj metav1.Object) error {
+    if true {
+        return nil
+    }
```
Comment (mrogers950) [inline pkg/operator/operator.go]:
```

                  _  /)
               < o / )
                 |/)\)
                  /\_
                  \__|=
                 (    )
                 __)(__
           _____/      \\_____
          |  _     ___   _   ||
          | | \     |   | \  ||
          | |  |    |   |  | ||
          | |_/     |   |_/  ||
          | | \     |   |    ||
          | |  \    |   |    ||
          | |   \. _|_. | .  ||
          |                  ||
          |    alfred        ||
          |                  ||
  *       | *   **    * **   |**      **
   \))..../.,(//,,..,,\||(,,.,\\,.((//
```

**Pull request merged**